docs/semgrep-cloud-platform/github-pr-comments.md

```diff
@@ -70,7 +70,7 @@ To enable dataflow traces feature in your CI pipeline, fulfill the following pre
 :::info Prerequisites
 - Enable GitHub pull request Semgrep comments.
```

Comment:
```suggestion
- Enable GitHub pull request Semgrep comments, as described on this page.
```
I didn't make this change in my proposal because I'm not sure if it's good practice, but I happened to come to this page via one of the lower sections, and was momentarily wondering "but where do I find the instructions to do this". So it could be helpful context, but if it's not something we normally do, then that's fine too.